projects/plugins/super-cache/js/admin.js

```diff
@@ -1,4 +1,7 @@
-jQuery( document ).ready( () => {
+/**
```

Comment:
I think the way it was handled before is better. The callback just needs to be an unnamed function, not an arrow function.

Instead of

```js
jQuery( document ).ready( () => {
```

use

```js
jQuery( document ).ready( function () {
```

That way, `handleMigrationButtons` won't be global.

Reply:
Thanks!

Reply:
@donnchawp I misled you when I said that it's better to use an unnamed function vs an arrow function. My intention was to avoid global scope for the `handleMigrationButtons` function.

Both get the job done. If there's a function inside an arrow function, it would still get scoped to the arrow function.